gtk/gtk-play.c

```diff
@@ -1253,6 +1299,8 @@ main (gint argc, gchar ** argv)
   gst_player_play (play.player);
   play.current_uri = g_list_first (play.uris);
 
+  g_timeout_add_seconds (5, (GSourceFunc) gtk_hide_toolbar_loop, &play);
```

Comment:
Here is why i have gtk_hide_toolbar_loop() running all the time, I was trying to handle the case where u go to full screen and then don't do any mouse activity, in that case we should simply hide the control. And similarly if you move the mouse and then stop moving  then also toolbar should get auto hide (maybe after 5sec ?). I think one thing which I should have done is track of keep track of time for last mouse movement and in gtk_hide_toolbar_loop() check if the time elapsed between last movement and the callback is greater than 5sec then hide the toolbar..
 Or i might be making is unnecessary complex ;)